hirte/src/client/client.c

```diff
@@ -13,6 +17,9 @@ static void signal_handler(int signal) {
 }
 
 int main() {
+        sd_bus *user_bus = user_bus_open(NULL);
+        service_call_shutdown(user_bus, ORCHESTRATOR_SERVICE_DEFAULT_NAME);
```

Comment:
Do we want to shut down the orchestrator here?

Reply:
Yes and no. In the future, the client is intended to be a CLI tool so that we could use it shutdown the orchestrator/node. For this we would need to extend it with CLI options etc. However, at the moment the client is a kind of place of examples (e.g. the signals and the infinity loop) and has 2nd prio. 
This change here was just to have an example. It can be removed if you want.

Reply:
Maybe comment it out and add a comment what it's for?  Or we could quickly add a ```getopt_long()``` here and put orch shutdown as a command line option.  Or just remove it.